packages/oracle-manager/test/integration/modules/OracleManagerModule/OracleManagerModule.nodes.ts

```diff
@@ -0,0 +1,44 @@
+import assert from 'assert/strict';
+import assertBn from '@synthetixio/core-utils/utils/assertions/assert-bignumber';
+import { ethers } from 'ethers';
+
+import { bootstrapWithNodes } from '../../bootstrap';
+import NodeTypes from '../../mixins/Node.types';
+import NodeOperations from '../../mixins/Node.operations';
+import assertRevert from '@synthetixio/core-utils/utils/assertions/assert-revert';
+
+describe('Oracle manager module nodes', function () {
```

Comment:
I should have come in earlier to comment on this ,but the format of these tests does not correspond to the v3 testing strategy cc @ajsantander . We will probably want to update these in the near future.